### PR TITLE
Add Mesosphere's Maven repository

### PR DESCRIPTION
**Pull request by gkleiman**

### CODE REVIEW

**Comment by aquamatthias** [comment]
:+1:

File: auth/build.sbt [hunk 1]
```diff
@@ -2,6 +2,8 @@ name := "auth-plugin"
 
 version := "1.0"
 
+resolvers += "Mesosphere Public Repo" at "http://downloads.mesosphere.io/maven"
```
Comment (janisz) [inline auth/build.sbt]: I think this repository is not open? [CI Failed](https://travis-ci.org/janisz/marathon-shiro-authentication-plugin/jobs/85485273)

**Comment by gkleiman** [comment]
@janisz: It is open, have you tried adding the following dependency?

`"mesosphere.marathon" %% "plugin-interface" % "0.12.0-RC1" % "provided"`

**Comment by janisz** [comment]
Oh, I have `mesosphere` instead of `mesosphere.marathon`. Now I have problem with play-json [version dependency](https://travis-ci.org/janisz/marathon-shiro-authentication-plugin/jobs/85529013)  but maybe when I change to RC release it will work. Thanks for your help

**Pull request merged**